pkgs/servers/jellyfin/default.nix

```diff
@@ -13,13 +13,13 @@
 
 buildDotnetModule rec {
   pname = "jellyfin";
-  version = "10.9.1"; # ensure that jellyfin-web has matching version
+  version = "10.9.2"; # ensure that jellyfin-web has matching version
```

Comment:
BTW you can add a `lib.warn` that will disturb ofborg's evaluation by comparing the versions.

Reply:
Or you can use an assertion.

Reply:
Thank you. Resolved by [this change](https://github.com/NixOS/nixpkgs/commit/ffb3ea67083513d5d7ffe1dfd5f6fdb7d0922178)!